test/imported.test.js

```diff
@@ -9,13 +9,70 @@ describe('cloudant imported features', function() {
   before(function() {
     IMPORTED_TEST = true;
   });
+
   after(function() {
     IMPORTED_TEST = false;
   });
-  require('loopback-datasource-juggler/test/datatype.test.js');
-  require('loopback-datasource-juggler/test/basic-querying.test.js');
-  require('loopback-datasource-juggler/test/hooks.test.js');
-  require('loopback-datasource-juggler/test/relations.test.js');
+
   require('loopback-datasource-juggler/test/include.test.js');
-  require('loopback-datasource-juggler/test/manipulation.test.js');
+  require('loopback-datasource-juggler/test/common.batch.js');
+});
+
+// Run the COUCHDB2 Test Suite.
+require('./init.js');
+
+process.env.COUCHDB2_TEST_SKIP_INIT = true;
+
+var skips = {
+  skip: [],
+};
+
+process.env.COUCHDB2_TEST_AUTOMIGRATE_SKIP = JSON.stringify(skips);
```

Comment:
There are tests skipped by Coucdb2. By sending in an empty skip array, all tests are run for Cloudant.